qiita_db/analysis.py

```diff
@@ -806,41 +818,90 @@ def create(cls, owner, name, description=None):
         description : str, optional
             Brief description of the collecton's overarching goal
         """
-        raise NotImplementedError()
+        conn_handler = SQLConnectionHandler()
+
+        sql = ("INSERT INTO qiita.{0} (email, name, description) "
+               "VALUES (%s, %s, %s)".format(cls._table))
+        conn_handler.execute(sql, [owner.id, name, description])
 
     @classmethod
     def delete(cls, id_):
         """Deletes a collection from the database"""
-        raise NotImplementedError()
+        conn_handler = SQLConnectionHandler()
+        if cls(id_).status == "public":
+            raise QiitaDBStatusError("Can't delete public study!")
```

Comment:
Is this message accurate? Seems like you are removing a study collection not a study.